### DISCUSSION TITLE
Tab 'bar' can be Top/Bottom in Horizontal config, Left/Right in Vertical config

### DISCUSSION THREAD

**Discussion by mjeveritt** [Chatterino/chatterino2#3855] [Ideas] [answered]
Web browsers often allow you to move the tab bar to the bottom of the screen rather than the top; likewise some users may find the layout more convenient on the left or right side according to it's position on the screen.

**Comment by Felanbird** [comment, marked as answer]
Already being implemented in #3847

**Comment by mjeveritt** [comment]
well, derp me .. didn't see in one list or the other, but now I know ! cheers!
